### DISCUSSION TITLE
Segment.io events not firing?

### DISCUSSION THREAD

**Discussion by asheinfeld** [vercel/next.js#16083] [Help]
I have a question regarding this example: https://github.com/vercel/next.js/tree/master/examples/with-segment-analytics

I changed the API key to use my own API from segment, but I don't see any events reaching segment.  I tried before copying this example on my own next.js app with the same results, so I went ahead and just copied/ran this project and I'm not seeing it working.

Has anyone successfully connected an SSR next.js app with Segment?

Thanks!

**Comment by jamesmosier** [comment]
I have Segment working with my Next.js project currently. Which events are you seeing not being fired? If you type `window.analytics` into your browsers console does the object exist?
